extensions/wikia/ArticleAsJson/ArticleAsJson.class.php

```diff
@@ -7,8 +7,15 @@ class ArticleAsJson extends WikiaService {
 		'imageMaxWidth' => false
 	];
 
+	const ICON_MAX_SIZE = 40;
 	const CACHE_VERSION = '0.0.3';
 
+	const ARTICLE_IMAGE = 'article-image';
+	const ARTICLE_VIDEO = 'article-video';
+	const GALLERY_IMAGE = 'gallery-image';
+	const ICON = 'icon';
+	const INFOBOX_IMAGE = 'infobox-image';
```

Comment:
I'm thinking about making the contexts prefixed by IMAGE_CONTEXT_ or something along similar lines. It'll be especially confusing when seeing ICON being used for the first time/